src/test/java/edu/cornell/kfs/sys/util/KualiDecimalXmlDifferenceEvaluator.java

```diff
@@ -0,0 +1,37 @@
+package edu.cornell.kfs.sys.util;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kuali.kfs.core.api.util.type.KualiDecimal;
+import org.xmlunit.diff.Comparison;
+import org.xmlunit.diff.ComparisonResult;
+import org.xmlunit.diff.DifferenceEvaluator;
+
+public class KualiDecimalXmlDifferenceEvaluator implements DifferenceEvaluator {
+    private static final Logger LOG = LogManager.getLogger();
+
+    @Override
+    public ComparisonResult evaluate(Comparison comparison, ComparisonResult outcome) {
+        if (outcome != ComparisonResult.EQUAL) {
+            if (comparison.getControlDetails() != null && comparison.getTestDetails() != null) {
+                String control = String.valueOf(comparison.getControlDetails().getValue());
+                String test = String.valueOf(comparison.getTestDetails().getValue());
```

Comment:
Is this comparison too broad in its current form? Should it narrow its focus to cases where text contents or attribute values were being compared?

Reply:
I tested differences in KualiDecimals and displayed the difference details as I expected, so I think the focus is sufficient

Reply:
Ok, thanks for confirming.